leetcode/581.shortest-unsorted-continuous-subarray.py

```diff
@@ -0,0 +1,28 @@
+class Solution(object):
+    def findUnsortedSubarray(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        n = len(nums)
+        l, i = 0, 0
+        for i in range(1, n):
+            if nums[i]<nums[i-1]:
+                l = i
```

Comment:
```
Input: [2,1]
Output: 0
Expected: 2
```
when you see a reverse trend, set l to i-1, r to i+1 because i-1 and i need to be reversed for the time being, so as the left boundary.